packages/core/src/controllers/relayer.ts

```diff
@@ -89,6 +89,13 @@ export class Relayer extends IRelayer {
     await Promise.all([this.messages.init(), this.transportOpen(), this.subscriber.init()]);
     this.registerEventListeners();
     this.initialized = true;
+    setTimeout(async () => {
+      if (this.subscriber.topics.length === 0) {
+        this.logger.info(`No topics subscribted to after init, closing transport`);
+        await this.transportClose();
+        this.transportExplicitlyClosed = false;
+      }
+    }, 10_000);
```

Comment:
```suggestion
    }, RELAYER_TRANSPORT_CUTOFF);
```
could we create a `const` in the constants/relayer.ts like
`export const RELAYER_TRANSPORT_CUTOFF = 10_000`
and import it into the relayer and the tests?
It would solve the need to update the value in multiple places when it gets changed

Reply:
Great suggestion, implemented